docs/SPEC.md

```diff
@@ -1724,9 +1724,9 @@ Histogram has the following properties:
 * `countColumn` string
     CountColumn is the name of the column in which to store the histogram counts.
     Defaults to `_value`.
-* `buckets` array of floats
-    Buckets is a list of upper bounds to use when computing the histogram frequencies.
-    Buckets should contain a bucket whose bound is the maximum value of the data set, this value can be set to positive infinity if no maximum is known.
+* `bins` array of floats
+    Bins is a list of upper bounds to use when computing the histogram frequencies.
+    Bins should contain a bin whose bound is the maximum value of the data set, this value can be set to positive infinity if no maximum is known.
```

Comment:
Tangential: this sentence is a little confusing.  Maybe `Each element in the array should contain a float value that represents the maximum value for a bin.`